hadoop-ozone/common/src/main/shellprofile.d/hadoop-ozone-manager.sh

```diff
@@ -19,8 +19,10 @@ if [[ "${HADOOP_SHELL_EXECNAME}" == ozone ]]; then
 fi
 
 _ozone_manager_hadoop_finalize() {
-  if [[ "${HADOOP_CLASSNAME}" == "org.apache.hadoop.ozone.om.OzoneManagerStarter" ]]; then
-   echo "Ozone Manager classpath extended by ${OZONE_MANAGER_CLASSPATH}"
-   hadoop_add_classpath "${OZONE_MANAGER_CLASSPATH}"
+  if [[ "${HADOOP_CLASSNAME}" == "org.apache.hadoop.ozone.om.OzoneManagerStarter" ]] &
+     [[ -n ${OZONE_MANAGER_CLASSPATH} ]];
+  then
+       echo "Ozone Manager classpath extended by ${OZONE_MANAGER_CLASSPATH}"
+       hadoop_add_classpath "${OZONE_MANAGER_CLASSPATH}"
```

Comment:
Nit: Indentation looks odd (7 spaces).  It should be 4.  (Although it was odd at 3 spaces before the change, too.)

Reply:
Thanks @adoroszlai  for the comment.
Fixed.